wasmer/instance.go

```diff
@@ -85,7 +85,7 @@ type Instance struct {
 	// The exported memory of a WebAssembly instance.
 	Memory MemoryHandler
 
-	Data        *int
+	Data        *uintptr
```

Comment:
is it neccessary to keep this as *uintptr - wouldn't be better to keep simply as uintptr ?